From: Jakub Wilk <jwilk@debian.org>
Date: Thu, 8 Oct 2015 11:57:05 -0700
Subject: Don't copy test suite output to a log.

When running the test suite, don't copy stdout and stderr output to a log
file, as this would break under LC_ALL=C, PYTHONWARNINGS=d and python3.X.

Forwarded: not-needed
Last-Update: 2011-09-18
---
 test/alltests.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/test/alltests.py b/test/alltests.py
index dbc206f..3e48ad6 100755
--- a/test/alltests.py
+++ b/test/alltests.py
@@ -80,9 +80,6 @@ def suite():
     return suite
 
 
-# must redirect stderr *before* first import of unittest
-sys.stdout = sys.stderr = Tee('alltests.out')
-
 import package_unittest  # noqa
 
 
